ApiDemos/kotlin/app/build.gradle

```diff
@@ -8,11 +8,11 @@ plugins {
 }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
     defaultConfig {
         applicationId "com.example.kotlindemos"
-        minSdkVersion 19
-        targetSdkVersion 31
+        minSdkVersion 21
```

Comment:
I don't think we're ready to bump up to minSDK 21 yet. Any reason you had to do this?

Reply:
No. This is set back to 19.